### PR TITLE
Bugfix/#61

### PR DESCRIPTION
**Pull request by AdamJ335**

### CODE REVIEW

**Comment by jrsmth** [comment]
Top bins!

**Review by jrsmth — APPROVED** [review]

**Review by haydende — CHANGES_REQUESTED** [review]
As commented on event_spec.py, I am concerned with how you've parameterised the tests.

**Review by haydende — APPROVED** [review]

File: src/app/model/event/event_spec.py [hunk 1]
```diff
@@ -3,79 +3,41 @@
 from src.app.model.event.event import Event
 
 
-class EventSpec:
+def extract_and_validate(file_name, expected_score, expected_streak):
```
[the remark this answers is not in the document]
Comment (AdamJ335) [reply src/app/model/event/event_spec.py:6]: Cleared up over further reflection. There is sufficient parameterisation of the given tests in `event_spec.py` No further changes required

[the remark this answers is not in the document]
Comment (haydende) [reply src/app/model/event/event_spec.py:6]: Please ignore this. I have completely misread what is happening here, and I can see that there nothing to be concerned with here. Sorry!

**Pull request merged**